internal/query/projection/user_test.go

```diff
@@ -78,10 +78,11 @@ func TestUserProjection_reduces(t *testing.T) {
 								domain.EmailAddress("email@zitadel.com"),
 								&sql.NullString{String: "+41 00 000 00 00", Valid: true},
 								true,
+								anyArg{},
```

Comment:
As the event payload doesn't carry any secret / password,  we can safely expect a `null` argument here, right?

Reply:
since we use sql.NullTime and check on equality, I had to update the tests with a testNow time